Directory.Build.props

```diff
@@ -5,7 +5,7 @@
     <Authors>$(Company)</Authors>
     <Copyright>Copyright © $(Company) $([System.DateTime]::Now.Year)</Copyright>
     <Trademark>$(Company)™</Trademark>
-    <VersionPrefix>6.1.0</VersionPrefix>
+    <VersionPrefix>6.2.0</VersionPrefix>
```

Comment:
1. We shouldn't be updating the version in a PR because multiple PRs could be combined for a single release.
2. You made a change to a `public interface` in this PR, which means a major version change - <https://semver.org/>.